src/components/book.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Form from './form';
+
+function Book() {
+  return (
+    <section className="Bookstore-CMS">
+      <section className="booklist">
+        <article>
+          <div>
+            <span>Action</span>
+            <h3>The Hunger Games</h3>
+            <span>Suzanne Collins</span>
+            <br />
+            <span>24% Completed</span>
+            <br />
+            <button type="button">Delete</button>
+          </div>
+          <br />
+          <div>
+            <span>Science Fiction</span>
+            <h3>Dune</h3>
+            <span>Suzanne Collins</span>
+            <br />
+            <span>50% Completed</span>
+            <br />
+            <button type="button">Delete</button>
+          </div>
+          <br />
+          <div>
+            <span>Economy</span>
+            <h3>Capital in the Twenty-First Century</h3>
+            <span>Suzanne Collins</span>
+            <br />
+            <span>64% Completed</span>
+            <br />
+            <button type="button">Delete</button>
+          </div>
+        </article>
+      </section>
+      <br />
+      <br />
+      <Form />
+    </section>
+  );
+}
```

Comment:
- [ ] As part of the project requirements please make sure to create components like a
  - `<BookList>` component that contains a list to render individual books.
  - Your `<Book>` component here should take the book data as a prop and renders the title and the author.
  - I'd also advise placing the form for adding a book in a `<BookForm>` component that renders a form with inputs for the title and the author of the book
as well as a button to submit.
  - And then finally you should create a `<Books>` component that will be the parent component for all the above components.